doc/developer-notes.md

```diff
@@ -625,10 +625,13 @@ member name:
 class AddressBookPage
 {
     Mode m_mode;
-}
+    int m_index;
 
-AddressBookPage::AddressBookPage(Mode _mode)
-    : m_mode(_mode)
+    AddressBookPage(Mode _m_mode, int _m_index)
+        : m_mode(_m_mode)
+        , m_index(_m_index)
```

Comment:
This is not stable with the https://github.com/bitcoin/bitcoin/tree/master/contrib/devtools#clang-format-diffpy script

Reply:
😢